src/serlo-editor/editor-ui/plugin-toolbar/plugin-tool-menu/anchor-link-copy-tool.tsx

```diff
@@ -0,0 +1,40 @@
+import { faHashtag } from '@fortawesome/free-solid-svg-icons'
+
+import { DropdownButton } from './dropdown-button'
+import { useEntityId } from '@/contexts/entity-id-context'
+import { useInstanceData } from '@/contexts/instance-context'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { showToastNotice } from '@/helper/show-toast-notice'
+
+interface AnchorLinkCopyToolProps {
+  pluginId: string
+}
+
+export function AnchorLinkCopyTool({ pluginId }: AnchorLinkCopyToolProps) {
+  const serloEntityId = useEntityId()
+  const editorStrings = useEditorStrings()
+  const { strings } = useInstanceData()
+
+  if (
+    !serloEntityId ||
+    !navigator.clipboard ||
+    !window.location.href.includes('add-revision')
+  )
+    return null
```

Comment:
I really dislike omitting curly braces when it goes to a new line like this, and I'd really appreciate adding curly braces here. The reason is that it's so easy to implement a bug by adding a line before the return statement line.

Or maybe:
```suggestion
  const isAddRevisionPage = window.location.href.includes('add-revision')
  if (!serloEntityId || !navigator.clipboard || !isAddRevisionPage) return null
```

But not a deal breaker, for sure.